src/emailLogin/runMigration.js

```diff
@@ -61,6 +61,10 @@ const getSiteAndContributors = async (site, dbClient) => {
       const userType = whitelistedDomains.filter((domain) => user.email.endsWith(domain)).length > 0 ? 'ADMIN' : 'CONTRIBUTOR';
       siteMemberValues.push(`(${userId}, ${repoId}, '${userType}')`);
     });
+    if (siteMemberValues.length === 0) {
+      console.log(`${site} has no CMS editors`);
+      return;
```

Comment:
may I get your help to output them to another file? 
Reason for this is because there are some redundant repos that have been created and no commits at all, was thinking we can take this opportunity to clean up our infra! 

what I am doing rn with Jackie is whatever repo that has no code, I am passing it to him to check if the folks still exist and if can I am deleting them. Perhaps we can do the same for repos with no CMS editors? 

PS. all repos have isomerAdmin as a collaborator, may I confirm that this if condition is indeed reachable?

Reply:
I'm actually filtering out all the isomer users already - so this is only to check which are the agency users to grant access to. What happens right now is

1. We retrieve the list of github ids mapped to the github team
2. We match the github ids to the appropriate isomer user (which means that agency users who have never logged in will have no matches here)
3. We add these people as site members

At the moment, I've been keeping track with the spreadsheet for any sites with this issue, but I'm thinking that this is a temporary measure - having no collaborators means that only our migration army has touched this repo so far, but I don't expect it to be this way closer to site launches, so I'm planning to rerun the script for these sites again on a later date.

I've also tried to keep track of any additional repos which have no code on the spreadsheet, though I've only found one so far - happy to chat more about this offline!

Reply:
> having no collaborators means that only our migration army has touched this repo so far,
Hmm not necessarily right, might have some users that use one email for github and another for isomer? 

> I've also tried to keep track of any additional repos which have no code on the spreadsheet, though I've only found one so far - happy to chat more about this offline! 
eh lets combine this, ill add another column in the main spreadsheet